src/assets/scss/components/_oktaSigninWidget.scss

```diff
@@ -175,8 +175,8 @@ sign-up-modal {
       }
     }
 
-    // Allow the account type dropdown to have rounded corners if the organisation field is hidden
-    &:has(+ .o-form-fieldset[style="display: none;"]) .chzn-container a {
+    // Allow the account type dropdown to have rounded corners if the organization field is hidden
```

Comment:
😂 `organisation` is spelt correctly.

Reply:
I was just switching to the American spelling for consistency in the rest of the repo and to match the input name `[data-se="o-form-input-organizationName"]`, but I can revert it.